src/lib.rs

```diff
@@ -423,18 +400,87 @@ pub mod pallet {
 		pub fn account_id() -> T::AccountId {
 			PALLET_ID.into_account()
 		}
+		/// The Account Id's balance
+		fn pot() -> BalanceOf<T> {
+			T::RewardCurrency::free_balance(&Self::account_id())
+		}
+		/// Make a vested payment
+		fn make_vested_payment(
```

Comment:
Some of the fn names are starting to confuse me. I always think of it "as a user calling this fn, what am I doing?" This fn is withdrawing, right?